client/app/private-database/configuration/private-database-configuration.html

```diff
@@ -1,76 +1,60 @@
 <div class="container-fluid px-0" data-ng-controller="PrivateDatabaseConfigurationsCtrl as configurationsCtrl">
 
+    <div data-ovh-alert="{{alerts.main}}"></div>
+
     <div class="text-center" data-ng-if="configurationsCtrl.loading">
         <oui-spinner data-size="l"></oui-spinner>
     </div>
 
-    <div class="row" data-ng-if="!configurationsCtrl.loading">
-        <div class="col-md-9">
-            <div data-ovh-alert="{{alerts.main}}"></div>
+    <div data-ng-if="!configurationsCtrl.loading">
 
-            <div class="alert alert-info" role="alert"
-                 data-i18n-static="privateDatabase_configurations_general_postgresql_soon"
-                 data-ng-if="configurationsCtrl.database.type === 'postgresql'">
-            </div>
+        <oui-message type="info"
+             data-ng-if="configurationsCtrl.database.type === 'postgresql'">
+            <span data-i18n-static="privateDatabase_configurations_general_postgresql_soon"></span>
+        </oui-message>
+        <oui-message type="info"
+             data-ng-if="configurationsCtrl.database.type !== 'postgresql' && configurationsCtrl.configurations.length === 0">
+            <span data-i18n-static="privateDatabase_capability_disabled_explanation"></span>
+        </oui-message>
 
-            <div data-ng-if="configurationsCtrl.database.type !== 'postgresql'">
-                <h2 data-ng-bind-html="tr('privateDatabase_configurations_general_' + configurationsCtrl.database.type)"></h2>
+        <div data-ng-if="configurationsCtrl.configurations.length >= 1">
 
-                <div class="alert alert-warning" role="alert"
-                     data-i18n-static="privateDatabase_configuration_alert"
-                     data-ng-if="configurationsCtrl.edit.value">
-                </div>
+            <oui-message type="warning">
```

Comment:
`data-type`

Reply:
No, the rules are to add `data-` to each attribute (cf https://github.com/ovh-ux/ovh-manager-web/pull/371#discussion_r177416164)